src/clib/lib/include/ert/enkf/enkf_config_node.hpp

```diff
@@ -28,10 +28,6 @@ struct enkf_config_node_struct {
      * Forward model? */
     bool forward_init;
 
-    /** Should this node be internalized - observe that question of what to
-     * internalize is MOSTLY handled at a higher level - without consulting
```

Comment:
The reasoning behind removing internalize needs its own commit message and some mention in the PR description.